corehq/apps/domain/models.py

```diff
@@ -943,9 +954,18 @@ def copies(self):
     def copies_of_parent(self):
         return Domain.view('domain/copied_from_snapshot', keys=[s._id for s in self.copied_from.snapshots()], include_docs=True)
 
-    def delete(self):
+    def delete(self, leave_tombstone=False):
+        if not leave_tombstone and not settings.UNIT_TESTING:
```

Comment:
why not make `leave_tombstone` default to `True`? is it just that you'd need to update all the unit tests?

Reply:
There was that, and also just that by far the more frequent call in our code base (because of tests) is `leave_tombstone=False`. It's called maybe twice anywhere with `leave_tombstone=True`, and I thought that that would be a nice thing to make explicit in those places for readability. (And then this line is to prevent new code from deleting domains in production _without_ also leaving a tombstone.)